index.html

```diff
@@ -10484,6 +10484,9 @@ <h3>User Prompt Handler</h3>
  <li><p>If <var>value</var> is a <a>string</a> set <var>value</var> to
  the <a data-cite=infra>map</a> «["<code>default</code>" → <var>value</var>]».
 
+ <li><p>If <var>value</var> is not an <a>Object</a> return <a>error</a>
```

Comment:
It can't be an Object here since we converted to infra values, but it could be an infra Map and indeed not being a Map would be an error.

Reply:
Good point! I completely overlooked that even through the map is used around those lines already.